backend/src/files/types/export-file.type.ts

```diff
@@ -0,0 +1,18 @@
+import { ApiProperty } from "@nestjs/swagger";
+
+export class ExportFileRequestBody {
+	@ApiProperty({ type: String, description: "export_type" })
+	exportType: "pdf" | "html" | "markdown";
```

Comment:
We can add validator here like this.

```typescript
@ApiProperty({ type: String, description: "export_type" })
@IsIn(["pdf", "html", "markdown"])
type: string;
```

This code allows us to avoid worrying about validating different input values within the business logic.